common/src/main/java/io/honeycomb/opentelemetry/EnvironmentConfiguration.java

```diff
@@ -181,11 +179,11 @@ public static void enableOTLPMetrics() {
         final String apiKey = getHoneycombMetricsApiKey();
         final String dataset = getHoneycombMetricsDataset();
 
-        if (!Strings.isNullOrEmpty(dataset)) {
+        if (dataset != null) {
```

Comment:
Is it worth duplicating the empty check as well?

https://github.com/google/guava/blob/master/android/guava/src/com/google/common/base/Platform.java#L58-L60

Reply:
Ah, yes of course! 👍  I've updated to use `isPresent`, which checks for both null and empty: `return value != null && !value.isEmpty();`

Reply:
Oh look at that, we already had one!